Problem2/UnitTestProblem2.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SimpleProblem3;
+namespace UnitTestProject1
+{
+    [TestClass]
+    public class UnitTestProblem2
+    {
+        [TestMethod]
+        
+            public void TwoLargestNumbersTest()
+        {
+            int[] a = new int[] { 0,2,4};
+            MethodProblem2 method = new MethodProblem2();
+            int[] actualMax =method.TwoLargestNumbers(a);
+            Assert.AreNotEqual(actualMax[0], int.MinValue, "Empty Array");
+            Assert.AreNotEqual(actualMax[1], int.MinValue, "Just One Element In Array");
+            int[] expectedMax = new int[] {4,2}; 
+            Assert.AreEqual(actualMax[0], expectedMax[0], "Different Max 1!");
+            Assert.AreEqual(actualMax[1], expectedMax[1], "Different Max 2!");
+           
+        }
+        
+    }
+}
```

Comment:
@OvidiuBan no worries. You'll get the hang of it. As to those asserts the fact remains that you can take out the first two and no matter what items you add to the array, all the cases where the test would pass with the asserts, will pass without them and all those which would fail would fail without them, making them obsolete.